public/app/plugins/datasource/elasticsearch/components/QueryEditor/QueryEditorRow.tsx

```diff
@@ -23,9 +22,9 @@ export const QueryEditorRow: FunctionComponent<Props> = ({
   const styles = getStyles(theme);
 
   return (
-    <fieldset className={styles.root}>
-      <div className={getInlineLabelStyles(theme, 17).label}>
-        <legend className={styles.label}>{label}</legend>
+    <InlineFieldRow>
```

Comment:
`FieldRow` was designed to be used with `InlineFields`, so it might not play well with using Labels directly.